deRSE23/README.md

```diff
@@ -0,0 +1,11 @@
+# deRSE23 Submission
+
+Abstract (and slides) for the [deRSE23](https://de-rse23.sciencesconf.org/) conference.
+
+* Title: Introducing the Fortran-lang community
+* Presenter: Ivan Pribec
+* Co-authors: Vincent Magnin, Asdrubal Lozada-Blanco, Ondrej Certik, Jeremie Vandenplas, Milan Curcic, Rohit Goswami, Sebastian Ehlert, Ioannis Nikiteas
+
+## Abstract
+
+A thriving community is vital for the success and continuity of a programming language. Despite Fortran’s role in science and engineering and the evolution of the language itself, attracting new users and retaining existing ones has been a challenge for Fortran. To address this, the open-source community Fortran-lang has formed in 2019. Here, we present the efforts of Fortran-lang such as the Fortran package manager, standard library, website, LFortran compiler, and other tools. Fortran-lang aims to provide a junction point for Fortran users, complementing the language standardization and compiler communities. Building on the best practices in software development, we aim to reduce the language learning curve and promote Fortran adoption.
```

Comment:
Nice, but I think "other tools" had the very concrete role of representing the VScode plugin and the language server, so if there is space for it I'd suggest to keep it in. :)

Reply:
By saying _we present the efforts including x, y, z_ it already implies there is more. I think "other tools" is just noise if you don't explicitly mention the VSCode and fortls.

Reply:
Oh sorry didn't really parse the "including" instead of "such as", my bad. I agree that either way the reference is very generic so I guess both ways are fine.

Reply:
Thanks, I've adopted some of the changes. I would like to mention fortls and the VSCode extension.